docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -123,18 +123,28 @@ Kubernetes 1.6 adds support for making use of custom metrics in the Horizontal P
 You can add custom metrics for the Horizontal Pod Autoscaler to use in the `autoscaling/v2alpha1` API.
 Kubernetes then queries the new custom metrics API to fetch the values of the appropriate custom metrics.
 
-### Prerequisites
-
-In order to use custom metrics in the Horizontal Pod Autoscaler, you must deploy your cluster with the
-`--horizontal-pod-autoscaler-use-rest-clients` flag on the controller manager set to true.  You must then configure
-your controller manager to speak to the API server through the API server aggregator, by setting the controller
-manager's target API server to the API server aggregator (using the `--apiserver` flag). The resource metrics API and
-custom metrics API must also be registered with the API server aggregator, and must be served by API servers running
-on the cluster.
-
-You can use Heapster's implementation of the resource metrics API by running Heapster with the`--api-server` flag set
-to true. A separate component must provide the custom metrics API (more information on the custom metrics API is
-available at [the k8s.io/metrics repository](https://github.com/kubernetes/metrics)).
+### Requirements
+
+To use custom metrics with your Horizontal Pod Autoscaler, you must set the necessary configurations when deploying your cluster:
+
+* [Enable the API aggregation layer](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/) if you have not already done so.
```

Comment:
Change this URL to a relative path, i.e.: `[Enable the API aggregation layer](/docs/tasks/access-kubernetes-api/configure-aggregation-layer/)`